uaclient/apt.py

```diff
@@ -626,20 +626,31 @@ def update_esm_caches(cfg) -> None:
 
     import apt  # type: ignore
 
+    from uaclient.actions import status
     from uaclient.entitlements.entitlement_status import ApplicationStatus
     from uaclient.entitlements.esm import (
         ESMAppsEntitlement,
         ESMInfraEntitlement,
     )
 
+    current_status = status(cfg)
```

Comment:
We only create the `status` cache if we run a command as root, but should we try reusing that cache here ? Or do you think this should be dynamic to always get the latest information available ?

Reply:
I added the call to the status cache first, only falling back to status (and creating the cache, because we will be root) in the case there is no cache in the first place.
I will create a subsequent PR to make sure this cache gets stale at some point.